hadoop/hadoop_cluster.json

```diff
@@ -116,7 +102,7 @@
         },
         "baseUri": {
             "type": "string",
-            "defaultValue": "https://raw.githubusercontent.com/deathly809/AzureStack-QuickStart-Templates/hadoop/hadoop",

```

Comment:
I meant rename the folder path to https://raw.githubusercontent.com/Azure/AzureStack-QuickStart-Templates/**hadoop-cluster**. It is also best to make this a parameter (_artifactsLocation) so that people can download all the artifacts and modify to suit.

Reply:
Don't quiet follow, could you point to an example?

Reply:
Follow this article: https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates-cloud-consistency#make-linked-templates-accessible-across-clouds

Reply:
Looks like the same thing as baseURI except they call it _artifactsLocation.  Changed it.